index.html

```diff
@@ -1,12 +1,67 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wanderrouten</title>
+    <title>Wanderregionen in Deutschland</title>
 </head>
+<header>
+    <nav>
+        <ul>
+            <li> <a href="#">Alpen</a></li>
+            <li><a href="#">Erzgebirge</a></li>
+            <li><a href="#">Schwarzwald</a></li>
+    </nav>
+</header>
+
 <body>
-    
+    <main>
+        <h1>Wanderrouten in Deutschand</h1>
+        <p>in Deutschland gibt es viele schöne Wanderrouten</p>
+        <section>
+            <h2>Wanderregionen Alpen</h2>
+            <p>Die Alpen sind das höchste Hochgebirge im Inneren Europas.</p>
+            <h3>Höchste Gipfel</h3>
+            <ul>
+                <li>Zugspitze</li>
+                <li>Mont Blanc</li>
+                <li>Matterhorn</li>
+            </ul>
+        </section>
+        <section>
+            <h2>Wanderregionen Erzgebirge</h2>
+            <p>Das Erzgebirge ist ein Mittelgebirge in Sachsen und Böhmen</p>
+            <h3>Höchste Gipfel</h3>
+            <ul>
+                <li>Große Fichtelberg</li>
+                <li>Auersberg</li>
+                <li>Katzenstein</li>
+            </ul>
+        </section>
+        <section>
+            <h2>Wanderregionen Schwarzwald</h2>
+            <p>Der Schwarzwald ist Deutschlands höchstes und größtes zusammenhängendes Mittelgebirge und liegt im
+                Südwesten Baden-Württembergs.</p>
+            <h3>Höchste Gipfel</h3>
+            <ul>
+                <li>Kandl</li>
+                <li>Schau ins land</li>
+                <li>Belchen</li>
+            </ul>
+        </section>
+    </main>
+    <footer>
+        <h4>Quellen</h4>
+        <nav>
+            <ul>
+                <li> <a href="https://de.wikipedia.org/wiki/Alpen">Alpen</a></li>
```

Comment:
```suggestion
                <li> <a href="https://de.wikipedia.org/wiki/Alpen">Alpen</a></li>
```<li><a href="https://de.wikipedia.org/wiki/Alpen">Alpen</a></li>